src/components/RootNavigator/BaseNavigator.js

```diff
@@ -139,11 +148,52 @@ export default function BaseNavigator() {
         component={AddAccountsNavigator}
         options={{ headerShown: false }}
       />
+      <Stack.Screen
+        name={NavigatorName.RequestAccount}
+        component={RequestAccountNavigator}
+        options={{
+          headerShown: false,
+        }}
+        listeners={({ route }) => ({
+          beforeRemove: () => {
+            const onError =
+              route.params?.onError || route.params?.params?.onError;
```

Comment:
a bit weird. let's have a comment to explain why we would have 2 cases.

Reply:
Added a comment